post/clang_tidy_review/clang_tidy_review/__init__.py

```diff
@@ -888,6 +931,8 @@ def get_line_ranges(diff, files):
 
     line_filter_json = []
     for name, lines in lines_by_file.items():
+        # On windows, only the line filters need backslashes
+        name = os.path.join(*name.split("/"))
```

Comment:
I don't quite see why this is necessary, is it because the diff always uses `/` even on Windows?

Reply:
Diffs yes but clang-tidy on windows will not work with `/`. Took a while to debug that.
Should I make the comment more clear?

Reply:
Ah yes, I can imagine that was a pain! Yes please :)